api/src/main/java/com/bjs/bjsapi/controllers/StudentScoreController.java

```diff
@@ -92,4 +95,36 @@ private double calculateExtra(int distance) {
 		return extra;
 	}
 
-}
\ No newline at end of file
+    public int studentAge(Student student) {
+        Date actual = new Date();
+        int studentAge = (int) student.getBirthDay().getYear();
+        int actualYear = (int) actual.getYear();
+        studentAge = actualYear - studentAge + 1900;
+        return studentAge;
```

Comment:
```java
Date current = new Date();
return (int) ChronoUnit.YEARS.between(current.toInstant(), student.getBirthDay().toInstant());
```

This code would make the calculation much easier and would not be deprecated.

Reply:
Nice

Reply:
```java
final LocalDate now = LocalDate.now();
final LocalDate birthDay = Instant.ofEpochMilli(student.getBirthDay().getTime())
    .atZone(ZoneId.systemDefault())
    .toLocalDate();

return (int) ChronoUnit.YEARS.between(birthDay,now);
```

Reply:
Okay, this will probably fit the requirements preferably.

```java 
final int birthYear = Instant.ofEpochMilli(student.getBirthDay().getTime()).atZone(ZoneId.systemDefault()).toLocalDate().getYear();
final int currentYear = Instant.now().atZone(ZoneId.systemDefault()).toLocalDate().getYear();
return Math.abs(birthYear - currentYear);
```

Reply:
Alright im doing some changes now